packages/tsx-editor/src/components/Editor.tsx

```diff
@@ -1,12 +1,19 @@
 import * as monaco from 'monaco-editor/esm/vs/editor/editor.api';
-import React from 'react';
+import * as React from 'react';
 import { IEditorProps } from './Editor.types';
 
-export class Editor extends React.Component<IEditorProps> {
-  private editor: monaco.editor.IStandaloneCodeEditor | undefined;
-  private editorRef = React.createRef<HTMLDivElement>();
+export const Editor = (props: IEditorProps) => {
```

Comment:
Just to be super explicit that this is a component:
```suggestion
export const Editor: React.FunctionComponent<IEditorProps> = (props: IEditorProps) => {
```